code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -388,9 +388,9 @@
 		msg += "<span class = 'deptradio'>Physical status:</span> <a href='?src=[UID()];medical=1'>\[[medical]\]</a>\n"
 		msg += "<span class = 'deptradio'>Medical records:</span> <a href='?src=[UID()];medrecord=`'>\[View\]</a> <a href='?src=[UID()];medrecordadd=`'>\[Add comment\]</a>\n"
 
-
-	if(print_flavor_text() && !skipface)
-		msg += "[print_flavor_text()]\n"
+	if(print_flavor_text() && get_organ("head"))
+		if(!(skipface || get_organ("head").disfigured || src.cloneloss > 50 || (HUSK in src.mutations)))
```

Comment:
One to many ()'s here. `src` is also implied so that can be removed.
The cloning damage is also already retrieved on line 268. It is however put in a temp var so it's (currently) not saved.

Reply:
I don't see where these unneeded parentheses are.

Reply:
Ah i just noticed the ! before it. All good for the parentheses!